packages/core/src/amazonq/errors.ts

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+import { ToolkitError } from '../shared/errors'
+
+export class CommonAmazonQContentLengthError extends ToolkitError {
```

Comment:
You can just call this
```suggestion
export class ContentLengthError extends ToolkitError {
```

I'd probably add a comment that's something like (you can improve the words):
```
/**
 * A shared error type for content length validation that can
 * be caught and processed by different agent implementations
 */
```

Reply:
Thanks!